src/components/Tree/TreeListItem.tsx

```diff
@@ -112,18 +112,19 @@ class TreeListItem extends UIComponent<TreeListItemProps> {
             defaultProps: {
               items,
               nested: true,
+              renderTitle,
```

Comment:
render title needs to be in the TreeListItem level. Different titles may need different renderers.

Reply:
It's a default prop, so it you will pass another renderer in `items` it will work:

```jsx
const items = [
{
    key: '2',
    title: ' two',
    items: [
      {
        key: '1',
        title: 'two one',
        renderTitle: OTHER_RENDERER,
      },
    ],
  },
]

const TreeExclusiveExample = () => <Tree items={items} renderTitle={DEFAULT_RENDERER} />
```